store/postgres/src/entities.rs

```diff
@@ -95,12 +121,47 @@ mod public {
     joinable!(entity_history -> event_meta_data (event_id));
     allow_tables_to_appear_in_same_query!(entity_history, event_meta_data);
 
+    /// We support different storage schemes per subgraph. This enum is used
+    /// to track which scheme a given subgraph uses and corresponds to the
+    /// `deployment_schema_version` type in the database.
+    ///
+    /// The column `deployment_schemas.version` stores that information for
+    /// each subgraph. Subgraphs that use the `Public` scheme have their
+    /// entities stored in the monolithic `public.entitites` table, subgraphs
+    /// that store their entities and history in a dedicated database schema
+    /// are marked with version `Split`.
+    ///
+    /// Migrating a subgraph amounts to changing the storage scheme for that
+    /// subgraph from one version to another. Whether a subgraph scheme needs
+    /// migrating is determined by `Table::needs_migrating`, the migration
+    /// machinery is kicked off with a call to `Connection::migrate`
     #[derive(DbEnum, Debug, Clone)]
     pub enum DeploymentSchemaVersion {
         Public,
         Split,
     }
 
+    /// Migrating a subgraph is broken into two steps: in the first step, the
+    /// schema for the new storage scheme is put into place; in the second
+    /// step data is moved from the old storage scheme to the new one. These
+    /// two steps happen in separate database transactions, since the first
+    /// step takes fairly strong locks, that can block other database work.
+    /// In the case of the `Public -> Split` migration, the schema changes
+    /// take a lock on `event_meta_data`, which, if held for too long, ends up
+    /// blocking any write access to the `public.entities` table because of
+    /// the way in which history is recorded. The second step, moving data,
+    /// only requires relatively weak locks that do not block write activity
+    /// to rows in `public.entities` that are not affected by the migration
+    /// of a different subgraph.
+    ///
+    /// The `Ready` state indicates that the subgraph is ready to use the
+    /// storage scheme indicated by `deployment_schemas.version`. After the
+    /// first step of the migration has been done, the `version` field remains
+    /// unchanged, but we indicate that we have put the new schema in place by
+    /// setting the `version` to `Tables`. At the end of the second migration
```

Comment:
This sentence is not parsing, it says that `version` remains unchanged and then that it is set to `Tables`.

Reply:
That second 'version' should have been 'state'. Fixed that.